enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/messaging/CoreReplicatedContentMarshal.java

```diff
@@ -119,6 +126,9 @@ public ReplicatedContent unmarshal0( ReadableChannel channel ) throws IOExceptio
             case DISTRIBUTED_OPERATION:
                 content = DistributedOperation.deserialize( channel );
                 break;
+            case DUMMY_REQUEST:
+                content = DummyRequest.Marshal.INSTANCE.unmarshal( channel );
+                break;
```

Comment:
We could refactor the byte identifiers to be an `enum` with the byte value as a field, and an `unmarshal` method. This switch block could then go away.